test/adagrams.test.js

```diff
@@ -145,18 +147,18 @@ describe("Adagrams", () => {
       const words = ["XXX", "XXXX", "X", "XX"];
       const correct = { word: "XXXX", score: scoreWord("XXXX") };
 
-      throw "Complete test by adding an assertion";
+      expect(highestScoreFrom(words)).toEqual(correct);
     });
 
-    describe("in case of tied score", () => {
-      const expectTie = (words) => {
-        const scores = words.map((word) => scoreWord(word));
-        const highScore = scores.reduce((h, s) => (h < s ? s : h), 0);
-        const tiedWords = scores.filter((s) => s == highScore);
+  describe("in case of tied score", () => {
```

Comment:
What is the outcome of this test scenario? I'd expect an `it` block to explain what the outcome is. Like, `describe("in case of tied score")` and then `it("returns more than one word")`